xlators/mount/fuse/src/fuse-bridge.c

```diff
@@ -7238,7 +7238,7 @@ struct volume_options options[] = {
     {
         .key = {"invalidate-limit"},
         .type = GF_OPTION_TYPE_INT,
-        .default_value = "0",
+        .default_value = "65536",
```

Comment:
We don't need to set the default value, a user can set the value at the time of mounting a volume after provide an argument invalidate-limit.